### PR TITLE
Optimism fee controller

### PR DESCRIPTION
**Pull request by Drypto13**

### CODE REVIEW

File: contracts/governance/buyback/FeeExtractAndDistribute_Optimism.sol [hunk 1]
```diff
@@ -0,0 +1,183 @@
+pragma solidity ^0.8.0;
+
+import "@openzeppelin-4.7.0/token/ERC20/IERC20.sol";
```
Comment (RomanHiden) [inline contracts/governance/buyback/FeeExtractAndDistribute_Optimism.sol]: this uses a version that exists only on  your machine. it does not compile for me

File: contracts/governance/buyback/FeeExtractAndDistribute_Optimism.sol [hunk 2]
```diff
@@ -0,0 +1,183 @@
+pragma solidity ^0.8.0;
+
```
Comment (RomanHiden) [inline contracts/governance/buyback/FeeExtractAndDistribute_Optimism.sol:8]: you should include the copyright

File: contracts/governance/buyback/FeeExtractAndDistribute_Optimism.sol (hunk 1 above)
Comment (Drypto13) [reply contracts/governance/buyback/FeeExtractAndDistribute_Optimism.sol]: fixed this.

**Review by RomanHiden — APPROVED** [review]
looks good @eugenesh1 please check as well

**Pull request merged**